### ISSUE TITLE
Incorrect calls to the game environment

### ISSUE THREAD

**Issue by jleni** [alexis-jacq/LOLA_DiCE#2]
There is a bug in the following lines:

https://github.com/alexis-jacq/LOLA_DiCE/blob/ec3f3f620a67df0a3e72f8e9227d09e4543dbb99/ipd_DiCE_om.py#L148-L150

https://github.com/alexis-jacq/LOLA_DiCE/blob/ec3f3f620a67df0a3e72f8e9227d09e4543dbb99/ipd_DiCE_om.py#L161-L163

In both cases, the agent object passes its action in the first position. 

This is correct when the call happens in the context of the first agent, however, this is incorrect for the second agent.

**Comment by jleni** [comment]
I will try to provide a PR later today

**Comment by alexis-jacq** [comment]
Yes, all agents are training to play in the first position, it would be incorrect for a non-symmetric game. I am working on a coin-game version (which is asymmetric) and I had to correct these calls. However, in the IPD, this would change nothing, as the game is symmetric.
